cypress/support/commands.js

```diff
@@ -63,6 +64,12 @@ Cypress.Commands.add('createArticle', (title, description, body) => {
       headers: {
         Authorization: `Token ${authToken}`
       }
+    }).then((response) => {
+      const article = {
+        slug: response.body.article.slug
+      };
+      const articleProperty = article;
+      cy.visit(`https://conduit.mate.academy/article/${articleProperty.slug}`);
```

Comment:
```suggestion
      cy.visit(`https://conduit.mate.academy/article/${article.slug}`);
```
you could use here this)